dependencies.gradle

```diff
@@ -0,0 +1,23 @@
+ext {
+    versions = [
+            androidSdk    : [
+                    compile   : 27,
+                    min       : 23,
+                    target    : 27,
+                    buildTools: '26.0.2'
+            ],
+            androidSupport: '27.0.0',
+            kotlin        : '1.1.51'
+    ]
+
+    libraries = [
+            build             : [
+                    androidGradle: 'com.android.tools.build:gradle:3.0.1',
+                    kotlinGradle : "org.jetbrains.kotlin:kotlin-gradle-plugin:${versions.kotlin}"
+            ],
+            jUnit             : 'junit:junit:4.12',
+            kotlinStdLibJre7  : "org.jetbrains.kotlin:kotlin-stdlib-jre7:${versions.kotlin}",
+            supportAnnotations: "com.android.support:support-annotations:${versions.androidSupport}",
+            supportAppCompatV7: "com.android.support:appcompat-v7:${versions.androidSupport}"
```

Comment:
same with android support libraries - they ship the whole set together so we don't want to mix-match versions